home/models.py

```diff
@@ -174,6 +168,25 @@ def get_context(self, request, *args, **kwargs):
         return context
 
 
+class FeaturedIncident(Orderable):
+    home_page = ParentalKey('home.HomePage', related_name='featured_incidents')
+    page = models.ForeignKey(Page, on_delete=models.CASCADE, related_name='+')
+
+    panels = [
+        PageChooserPanel('page', ('incident.IncidentPage',)),
+    ]
+
+
+class FeaturedBlogPost(Orderable):
+    home_page = ParentalKey('home.HomePage', related_name='featured_blog_posts')
+    page = models.ForeignKey(Page, on_delete=models.CASCADE, related_name='+')
```

Comment:
Something that @chigby pointed out to me in my blog index page PR and I think applies here as well is we can just use `blog.BlogPage` here itself and not mention in PageChooserPanel. Same applies to `FeaturedIncident` above.